### DISCUSSION TITLE
🎉 Open Tech Summit - I.I.I.T Hyderabad

### DISCUSSION THREAD

**Discussion by imabp** [Tech-Phantoms/community#113] [Announcements]
👋 Hello Community Members and @Tech-Phantoms/staff 

We are happy to announce that we will be having few open source projects  from our organisation, to Open Tech Summit. organised by [OSDG Group at I.I.I.T Hyderabad](https://osdg.iiit.ac.in) 

Also meet Kshitijaa ( @deutranium ), she is the overall coordinator of Open Source Developers Group, IIIT Hyderabad. She will be our go-to person, for the queries and more information related to it. 
_But it is recommended to going through brochure (attached below) before asking questions_ 

---
### OKRs and KPIs
**The objectives OKRs for us as an Open Organisation from the event are:**

1. Increase the meaningful contribution and pave the way for beginners to contribute to an existing project.

2. Give a boost to [Tech-Phantoms/Maintainers-Program](https://github.com/Tech-Phantoms/Maintainers-Program)
    - developing GitHub actions for maintainers program.
    - bringing in more maintainers for the program
    
3. Onboarding members from OSDG Group & Network from IIIT Hyderabad to our [Discord](https://discord.gg/wkPBrQb7pt), so our community members can take maximum benefits and build a good network cc: @deutranium 

Brochure: Also you can view [Open Tech Summit Brochure](https://cdn.discordapp.com/attachments/903223090274775051/913839471554887740/Open_Tech_Summit_Community_Partners.pdf)

---

**Keep an eye on the discussion, we will be updating with selected projects with features.**

cc: @Souvikns @kalzen15 @syshcndr @Panquesito7 @deutranium

**Comment by deutranium** [comment]
Glad to have you on board!
